static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.ts

```diff
@@ -109,6 +109,7 @@ export const renderStickyScrollForMoreLabel = (
 		const scrollForMoreLabel = document.createElement('div');
 		scrollForMoreLabel.classList.add('ad-slot__scroll');
 		scrollForMoreLabel.innerHTML = 'Scroll for More';
+		scrollForMoreLabel.setAttribute('role', 'button');
```

Comment:
nice a11y improvement 👍

Reply:
Could we make it a `<button>` instead of a `<div>`?